samples/json.js

```diff
@@ -0,0 +1,17 @@
+import http from "k6/http";
+import { check } from "k6";
+
+export default function() {
+    // Send a JSON encoded POST request
+    let body = JSON.stringify({ key: "value" });
+    let r = http.post("http://httpbin.org/post", body, { headers: { "Content-Type": "application/json" }});
+
+    // Use JSON.parse to deserialize the JSON (instead of using the r.json() method)
```

Comment:
@ragnarlonn Yes, `Response.json()` is available, see https://github.com/loadimpact/k6/blob/master/js/lib/k6/http.js#L15.